README.md

```diff
@@ -197,6 +200,12 @@ The runtime circumstances and algorithm for getting an ad or not, are not in the
 
 **Note**: The SDK only operates with an active network connection; you will not be able to do anything without it.
 
+### Tutorials
+
+...
+
+> [Tutorials](/tutorials/)
```

Comment:
There is a link on tutorials in line 177. If you want a separate Tutorials section, I think you should just move that text instead of that single word

Reply:
@MarkGerl this is valid point.

Reply:
I wasn't done moving this around.

Reply:
I wanted to give Tutorials their own section in the index.  Even if only one sentence and the link.